CMakeLists.txt

```diff
@@ -46,6 +46,7 @@ option(OLP_SDK_ENABLE_DEFAULT_CACHE "Enable default cache implementation based o
 option(OLP_SDK_ENABLE_DEFAULT_CACHE_LMDB "Enable default cache implementation based on LMDB" OFF)
 option(OLP_SDK_ENABLE_ANDROID_CURL "Enable curl based network layer for Android" OFF)
 option(OLP_SDK_ENABLE_IOS_BACKGROUND_DOWNLOAD "Enable iOS network layer downloading in background. Under testing." OFF)
+option(OLP_SDK_ENABLE_OFFLINE_MODE "Enable offline mode. All network requests returned with error." OFF)
```

Comment:
Include here that network-related dependencies are not needed as well.

Reply:
Extended